pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1276,7 +1276,7 @@ func (n *NetworkPolicyController) syncAppliedToGroup(key string) error {
 				if podSet == nil {
 					podSet = controlplane.GroupMemberSet{}
 				}
-				podSet.Insert(podToGroupMember(pod, false))
+				podSet.Insert(podToGroupMember(pod, true))
```

Comment:
cc @jianjuns for awareness as he suggested to query interface cache directly to get IPs of members in AppliedToGroup, this update changes it. and you perhaps need to justify why it's needed.

Reply:
I have tried to overwrite the IPs of members on pkg/agent/controller/networkplicy code but I found that the AppliedToGroup is sent from pkg/controller/networkpolicy code. Is there an another way to query the IPs of members on  pkg/agent/controller/networkplicy code?

Reply:
Is the comment added at the right file/line? I do not see how the code is relevant to "query interface cache".

Reply:
This changed has been reverted.